example/site/index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
```

Comment:
I think this page is a great addition, just two adjustments I'd suggest:

1. Just style it without adding the material components library, not worth the addition
2. It should show on first load, but probably not every time

Thoughts?

Reply:
> Just style it without adding the material components library, not worth the addition

It is just a development splash page. No use managing styling on the page. Any library can be used if you want to use a Netlify compiled style.

> It should show on first load, but probably not every time

Don't understand what you mean by this? Every time?